packages/zoe/test/unitTests/contracts/sellTickets.test.js

```diff
@@ -200,7 +200,9 @@ test(`mint and sell opera tickets`, async t => {
     const {
       value: [{ instance }],
     } = await E(invitationIssuer).getAmountOf(invitation);
+    /** @type {any} */
     const ticketSalesPublicFacet = await E(zoe).getPublicFacet(instance);
```

Comment:
what happened to `getPublicFacet` to necessitate all these casts? I should take another look.